directory_sso_api_client/client.py

```diff
@@ -14,3 +15,7 @@ def __init__(self, base_url=None, api_key=None):
 
     def ping(self):
         return self.get(url=self.endpoints['ping'])
+
+    def get_user_by_email(self, email):
```

Comment:
this is redundant, although I suspect this is an artefact of an unfinished refactor.

Reply:
actually this is the correct one, the other one was redundant. I just removed it.